app/source/components/notifications/notifications.jsx

```diff
@@ -28,6 +30,13 @@ class Notifications extends React.Component {
     this.setState({ volume });
   };
 
+  handleNewNotification = () => {
+    this.play();
```

Comment:
Weblyd på native notification?

Reply:
nei, men ved de 'vanlige' notifikasjonene på /notifications vil jo spilles av om ikke lyden er muted?

Reply:
De spilles vel av fra før?

Reply:
Blitt flytta på, renamet shouldPlayAudio() til newNotification() (shitty navn?) hvor handleNewNotification() tar seg av hva som skal skje om det har kommet nye notifikasjoner

Reply:
Kanskje den skal ha et ikke-shitty navn? :) Jeg tror også `shouldPlayAudio` og `shouldShowNativeNotification` må håndteres separat siden det er to helt separate ting

Reply:
Om jeg har skjønt det riktig så er alt det shouldPlayAudio gjør er å sammenligne to lister med notifikasjoner, og hvis første elementet har forandret seg så returnerer den true?

Og det er vel når denne er true at man skal spille av lyd / spawne native notifikasjon? derav, handleNewNotification om listen er forandret

Reply:
Ah. Da burde funksjonen som sammenligner listene hete `hasNewNotification` eller noe sånt da

Reply:
oppdatert!